src/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -11,7 +11,9 @@ internal static partial class Libraries
         internal const string HttpNative = "System.Net.Http.Native";
         internal const string NetSecurityNative = "System.Net.Security.Native";
         internal const string CryptoNative = "System.Security.Cryptography.Native.OpenSsl";
+#if !NO_RID
```

Comment:
Is it worth ifdefing a constant in a test library?

Reply:
If I don't add this ifdef here I would have duplicate constants as both Windows/Interop.Libraries.cs and Unix/Interop.Libraries.cs are now compiled. I know this is ugly but it doesn't look like these interop files were designed with RID-less configurations in mind.

Reply:
Interesting.  That actually means you need two different constants if you would use these, otherwise you'd get the wrong value on Unix.

What did you need out of this file?  It might make more sense to duplicate it it is just a single thing.

The main thing that caught my eye here was the double negative in the ifdef "not no", then the odd subsetting that was going on.